src/main/java/org/junitpioneer/jupiter/issue/IssueExtensionExecutionListener.java

```diff
@@ -88,7 +87,7 @@ public void testPlanExecutionFinished(TestPlan testPlan) {
 			return;
 
 		List<IssueTestSuite> issueTestSuites = createIssueTestSuites();
-		for (IssueProcessor issueProcessor : ServiceLoader.load(IssueProcessor.class)) {
+		for (IssueProcessor issueProcessor : IssueProcessorRegistry.getIssueProcessors()) {
```

Comment:
I don't like how these purely test-related problem bleeds into the production code.